packages/database/src/repository/User.ts

```diff
@@ -203,10 +196,7 @@ export function UserRepository(database?: DataSource): Repository {
     // future support for multiple mods
     const addedModuleCodes = [moduleCode]
     // 1. Return empty array if module in modulesDone or modulesDoing
-    await UserRepository(db).loadRelations(user, {
-      modulesDone: true,
-      modulesDoing: true,
-    })
+    await UserRepository(db).findOneById(user.id)
```

Comment:
should use copy

currently this passes tests because the only function that uses this loads relations, but we might want to have this function as an endpoint